rmgpy/rmg/main.py

```diff
@@ -1247,7 +1247,7 @@ def makeSeedMech(self,firstTime=False):
                     q += 1
                 self.name = name + str(q)
         
-        seedDir = os.path.join(self.outputDirectory,'seed')
+        seedDir = os.path.join(self.outputDirectory, 'seed')
```

Comment:
This change should be in a separate commit

Reply:
Change not included in this PR since I do not change this line. Thanks for the catch!